app/webpack.config.js

```diff
@@ -187,6 +188,12 @@ module.exports = {
             },
             canPrint: true,
         }),
+        new ImageminPlugin({
+            disable: !isProduction(), // Disable during development
+            pngquant: {
+                quality: '30-50',
```

Comment:
This seems quite low, have you played around much with this value? I'd be more comfortable with a conservative quality pullback.

Reply:
@fonty1 I didn't do an exhaustive test but didn't see a significant quality difference between original and optimised for any of the images I tested. I'll test a bit more.